### PR TITLE
Added Photo and Blog

### PR DESCRIPTION
**Pull request by samyakbvs**

### CODE REVIEW

**Comment by gabru-md** [comment]
Please give us a link to your changes.

**Comment by samyakbvs** [comment]
What do you mean?

**Comment by samyakbvs** [comment]
What do you mean?
@gabru-md

**Comment by samyakbvs** [comment]
Please can we go ahead without a test link
Just a request :)

**Comment by anshumanv** [comment]
Hey @samyakbvs there have been a lot of PRs from you and none of them had a test link, please provide us a test link so we can verify the changes quickly from now on. I hope you understand and take this in a positive way.

**Comment by supunTE** [comment]
Give a link for see changes

**Comment by samyakbvs** [comment]
OKay but please explain to me how to do it

**Comment by samyakbvs** [comment]
@anshumanv

**Comment by anshumanv** [comment]
Follow - https://pages.github.com/

**Comment by samyakbvs** [comment]
@anshumanv I have gone through that page several times but I could find nothing so can you pls explain to me how its done

**Comment by anshumanv** [comment]
It's simple read and follow - https://help.github.com/articles/configuring-a-publishing-source-for-github-pages/#enabling-github-pages-to-publish-your-site-from-master-or-gh-pages

Please tell me which line is difficult to understand. :smile:

**Comment by anshumanv** [comment]
image

**Comment by anshumanv** [comment]
image

**Comment by anshumanv** [comment]
image

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/gci17.fossasia.org/
Is this the one?

**Comment by anshumanv** [comment]
Yes, exactly. See I told you it's easy and you were worried about it for no reason. :smile:

**Comment by samyakbvs** [comment]
Thank you so much

**Comment by samyakbvs** [comment]
now can the PR merge?

**Comment by samyakbvs** [comment]
@anshumanv

**Comment by anshumanv** [comment]
No, Let others review, provide feedback, they will merge when they feel apt.
Also, we're unsure if we need the interview in this form so please wait for a while.

**Comment by samyakbvs** [comment]
Okay no problem

**Comment by samyakbvs** [comment]
Till then should I abandon the task and work on another one

**Comment by samyakbvs** [comment]
Roughly how much time will it take for more people to review?

**Comment by samyakbvs** [comment]
@anshumanv

**Comment by anshumanv** [comment]
From my understanding, since the task says either a video or a blog, I think this should work.

File: _config.yml
Comment (anshumanv) [inline _config.yml]: Remove this file

**Review by anshumanv — CHANGES_REQUESTED** [review]

**Comment by samyakbvs** [comment]
Done!

**Comment by samyakbvs** [comment]
@anshumanv

**Review by abishekvashok — APPROVED** [review]
LGTM, but if you could squash the commits, it will be better.

**Review by akshatnitd — CHANGES_REQUESTED** [review]
The demo link you provided isn't working. Can you please check it.

Also, please squash the commits. Refer https://gist.github.com/patik/b8a9dc5cd356f9f6f980 for help.

**Comment by samyakbvs** [comment]
https://samyakbvs.github.io/gci17.fossasia.org/

**Comment by samyakbvs** [comment]
The link is working perfectly fine now

**Comment by samyakbvs** [comment]
@anshumanv  Please excuse me for the last time, I will squash the commits next time positively. I am not able to do them now as I didn't work on Github desktop

**Comment by samyakbvs** [comment]
next time for sure you will see squashed commits

**Comment by samyakbvs** [comment]
I hope you understand :)

**Comment by anshumanv** [comment]
Okay, no issues about squashing commits if you cant do it this time. 
Please wait for others to review.

**Review by ParthS007 — APPROVED** [review]
LGTM !

**Review by anshumanv — APPROVED** [review]

**Comment by anshumanv** [comment]
@ParthS007 should we merge or wait?

**Comment by ParthS007** [comment]
I think we should wait for @akshatnitd  approval. BTW it's fine!

**Comment by anshumanv** [comment]
Aight, cool. Let's wait for him to complete review.

**Comment by samyakbvs** [comment]
@akshatnitd  Please see if it is fine now?

**Comment by abishekvashok** [comment]
@samyakbvs can you squash rest looks fine.

**Comment by harshit98** [comment]
@samyakbvs I have observed you many times, you don't keep patience.

> Roughly how much time will it take for more people to review?

Mentors will review your work as soon as they will get time.  You should respect their time as well.

Since a lot of pull request of yours has been merged, I think it's time you should learn squashing of commits here. Keep up the good work! 💯 

@abishekvashok @anshumanv @ParthS007 Please help him out. Thanks!

**Comment by samyakbvs** [comment]
Sorry, @harshit98 will take care of that from next time and now I know that how to squash commits but I have done my work directly on the site so I am not able to squash the commits, that's why I am saying that Pls excuse me for the last time. Also, I am sorry again

**Comment by harshit98** [comment]
Okay @samyakbvs. Next time, please squash your commits. I'm merging this pull request. 👍

**Comment by samyakbvs** [comment]
Thanks

**Pull request merged**
